src/components/progress/base.ts

```diff
@@ -56,18 +56,7 @@ export abstract class IgcProgressBaseComponent extends LitElement {
   public labelFormat!: string;
 
   @watch('indeterminate', { waitUntilFirstUpdate: true })
-  protected indeterminateChange() {
-    if (!this.indeterminate) {
-      this.runAnimation(0, this.value, true);
-    } else {
-      setTimeout(() => {
-        this.animation?.finish();
-        setTimeout(() => {
-          this.animation?.cancel();
-        }, 0);
-      }, 0);
-    }
-  }
+  protected indeterminateChange() {}
```

Comment:
I think you can implement the cancel of the indeterminate animation in the base class itself.
Something along the lines of:
```suggestion
  @query('[part~="indeterminate"]')
  protected indeterminateElement!: Element;
  
  protected cancelIndeterminate() {
    this.indeterminateElement.getAnimations().forEach(animation => animation.cancel());
  }
  
  protected indeterminateChange() {
    //...implementation goes here
  }
```